docs/docs/admin/installation.mdx

```diff
@@ -186,3 +186,92 @@ Then point your Ingress to the Anubis port:
               # diff-add
               name: anubis
 ```
+
+## Nginx
+
+The below is a very minimal Nginx config example to demonstrate how to
+insert Anubis in the middle of a standard nginx configuration.
+Do not forget to add `access_log`, `server_tokens`, etc. as usual.
+Also one does have to correct the paths, eg for SSL keys/certs.
+
+First we define `conf-anubis.inc` where we include standard Anubis configuration:
+```
+# Serve Anubis images directly instead of having it pass through to the service
+# Place `happy.webp`, `pensive.webp` and `reject.webp` in the directory.
+location /.within.website/x/cmd/anubis/static/img/
+{
+        alias /www/anubis/www/img/;
+}
+
+# Forward to anubis
+location /
+{
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_pass http://anubis;
+}
+```
+
+Then we define a Anubis backend as `server-anubis.conf`, which should
+be loaded from /etc/nginx/nginx.conf with a `include server-*.conf`.
+```
+upstream anubis
+{
+        # Try anubis first over a UNIX socket
+        server unix:/run/anubis/nginx.sock;
+	# Use the HTTP variant if you have the BIND on TCP:
+	# server http://127.0.0.1:8923;
+
+        # Fall back to avoid anubis when it is unavailable
+        # Especially useful when the socket is down
+        # as the service did not start or has other issues
+        server unix:/run/nginx.sock backup;
```

Comment:
If the socket is down then anubis is not working, maybe it crashed, but that is not a useful default configuration as a normal administrator would not notice it in time, even when visiting the size because they likely already had the cookie cached and the site would exposed to bots.